<launch>
  <node name="dynamixel_workbench_multi_port" pkg="dynamixel_workbench_controllers" type="dynamixel_workbench_multi_port" required="true" output="screen">
    <rosparam>
      profile_velocity: 100
      profile_acceleration: 20
      pan_motor:
        device_name: "/dev/ttyUSB0"
        baud_rate: 1000000
        motor_model: "XM430_W210_R"
        protocol_version: 2.0
        motor_id: 1
      tilt_motor:
        device_name: "/dev/ttyUSB1"
        baud_rate: 1000000
        motor_model: "RX_28"
        protocol_version: 1.0
        motor_id: 2
    </rosparam>
  </node>
</launch>
